src/options.js

```diff
@@ -66,6 +66,7 @@ function normalize(options) {
     // for a few versions. This code can be removed later.
     normalized.trailingComma = "es5";
 
+    // eslint-disable-next-line no-console
     console.warn(
```

Comment:
Not sure what to do with these as they aren't part of the CLI. In the naught corner for now.

Reply:
Maybe we can pass an internal argument `logger` to this function, and defaults to `console`?